src/SME.SGP.Infra/Consts/ArquivoContants.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace SME.SGP.Infra
+{
+    public static class ArquivoContants
+    {
+        public const string PastaTemporaria = "/temp/";
+        public const string PastaAquivos_E_Temporaria = "/Arquivos/temp/";
```

Comment:
Usamos o padrão Pascal Case nos nomes de variáveis
```suggestion
        public const string PastaAquivosTemporaria = "/Arquivos/temp/";
```